include/RAJA/pattern/WorkGroup/WorkStorage.hpp

```diff
@@ -621,72 +629,75 @@ class WorkStorage<RAJA::ragged_array_of_objects, ALLOCATOR_T, Dispatcher_T>
   void move_assign_private(WorkStorage&& rhs, std::false_type)
   {
     clear();
-    if (m_aloc == rhs.m_aloc) {
+    if (m_aloc == rhs.m_aloc)
+    {
 
       m_offsets     = std::move(rhs.m_offsets);
       m_array_begin = rhs.m_array_begin;
-      m_array_end   = rhs.m_array_end  ;
-      m_array_cap   = rhs.m_array_cap  ;
+      m_array_end   = rhs.m_array_end;
+      m_array_cap   = rhs.m_array_cap;
 
       rhs.m_array_begin = nullptr;
       rhs.m_array_end   = nullptr;
       rhs.m_array_cap   = nullptr;
-    } else {
+    }
+    else
+    {
       array_reserve(rhs.storage_size());
 
-      for (size_type i = 0; i < rhs.size(); ++i) {
+      for (size_type i = 0; i < rhs.size(); ++i)
+      {
         m_array_end = m_array_begin + rhs.m_offsets[i];
         move_destroy_value(m_array_end, rhs.m_array_begin + rhs.m_offsets[i]);
         m_offsets.emplace_back(rhs.m_offsets[i]);
       }
-      m_array_end = m_array_begin + rhs.storage_size();
+      m_array_end     = m_array_begin + rhs.storage_size();
       rhs.m_array_end = rhs.m_array_begin;
       rhs.m_offsets.clear();
       rhs.clear();
     }
   }
 
   // get loop storage capacity, used and unused in bytes
-  size_type storage_capacity() const
-  {
-    return m_array_cap - m_array_begin;
-  }
+  size_type storage_capacity() const { return m_array_cap - m_array_begin; }
 
   // get unused loop storage capacity in bytes
-  size_type storage_unused() const
-  {
-    return m_array_cap - m_array_end;
-  }
+  size_type storage_unused() const { return m_array_cap - m_array_end; }
 
   // reserve space for loop_storage_size bytes of loop storage
   void array_reserve(size_type loop_storage_size)
   {
-    if (loop_storage_size > storage_capacity()) {
+    if (loop_storage_size > storage_capacity())
+    {
```

Comment:
Looks like we decided to put the curly on the line after conditionals.

Reply:
Yes. John discussed this in a recent meeting.